cli/lib/index.js

```diff
@@ -13,6 +13,11 @@ const logger = new ColorfulChalkLogger('kamus-cli', {
   colorful: true, // the default value is true.
 }, process.argv);
 
+// ColorfulChalkLogger takes care of the verbosity, we don't want to pass it to caporal
+if (process.argv.indexOf('--verbose')) {
```

Comment:
Maybe we need to remove that also from the help? Is that possible?

Reply:
`--verbose` is still operational, we just take care of that in the `ColorfulChalkLogger` and not in caporal.

Reply:
You have a point here, we can't remove the --verbose since it's internal to caporal, but we can set the `--log-level` of `ColorfulChalkLogger` to `verbose` when users set it.
I'll add a commit.